frontend/public/manifest.json

```diff
@@ -0,0 +1,15 @@
+{
+  "short_name": "React App",
```

Comment:
we should use the gitbase-playground values :dancer:

Reply:
Done in 3d57d81.